docs/api-reference.md

```
@@ -833,6 +840,82 @@ new AuthorizationPermissionGrantBuilder()
+buildRawXmlString(document: XmlObject, options?: { pretty?: boolean }): string
+```
+Low-level escape hatch for pre-shaped `XmlObject` inputs (e.g. produced by `parseXml`). Returns a `string` — wrap with `Buffer.from(xml, 'utf8')` for byte-ready output.
+
+```ts
+parseXml(xml: string): XmlDocument
+buildXml(document: XmlDocument): string
+stripBom(text: string): string
+ORDER_MAP: Record<string, string[]>
+comparePKey(a: string, b: string): number
+orderXmlObject(obj: XmlObject, parent: string): XmlObject
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)